src/ui/server/constants/regex.ts

```diff
@@ -14,6 +14,13 @@ export const REGEX = {
    */
   INCLUDES_NUMERICAL_CHARACTERS: /\d+/g,
 
+  /**
+   * NUMERICAL_CHARACTERS:
+   * Regex that checks a string includes 0-9 characters without the global flag.
+   * Note: \d is exactly the same as [0-9]
+   */
+  NUMERICAL_CHARACTERS: /^\d+$/,
```

Comment:
I think this should be called `ONLY_NUMERICAL_CHARACTERS`, and documentation updated to say "only" instead of "includes", because this only checks if a string has pure numerical characters rather than if a string includes numerical characters